### PR TITLE
Dev robot status

### PR DESCRIPTION
**Pull request by JeremieBourque1**

### CODE REVIEW

File: opentera_webrtc_demos/launch/demo.launch [hunk 1]
```diff
@@ -62,5 +62,8 @@
         <arg name="input_topic" value="/webrtc_data"/>
     </include>
 
+    <!-- Robot status simulated data -->
+    <node name="robot_status" type="robot_status.py" pkg="opentera_webrtc_demos" />
+
 
 </launch>
```
Comment (doumdi) [inline opentera_webrtc_demos/launch/demo.launch:69]: Add newline at the end.

File: opentera_webrtc_demos/scripts/robot_status.py [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+                r.sleep()
+
+
+if __name__ == '__main__':
+    print("Robot Status Ready")
+    try:
+        robot_status = RobotStatus()
+        robot_status.run()
+    except rospy.ROSInterruptException:
+        pass
```
Comment (doumdi) [inline opentera_webrtc_demos/scripts/robot_status.py:33]: Add newline at the end.

File: opentera_webrtc_demos/launch/demo.launch (hunk 1 above)
Comment (JeremieBourque1) [reply opentera_webrtc_demos/launch/demo.launch]: Added

File: opentera_webrtc_demos/scripts/robot_status.py (hunk 2 above)
Comment (JeremieBourque1) [reply opentera_webrtc_demos/scripts/robot_status.py]: Added

**Review by doumdi — APPROVED** [review]

**Pull request merged**